src/components/forms/SmallGrantsForm.tsx

```diff
@@ -1717,6 +1768,77 @@ export const SmallGrantsForm: FC = () => {
             </Fade>
           </Box>
 
+          <Controller
+            name='howDidYouHearAboutESP'
+            control={control}
+            defaultValue={{ value: '', label: '' }}
+            rules={{ required: true, validate: selected => selected.value !== '' }}
+            render={({ field: { onChange }, fieldState: { error } }) => (
+              <FormControl id='how-did-you-hear-about-ESP-control' isRequired mb={8}>
+                <FormLabel htmlFor='howDidYouHearAboutESP'>
+                  <PageText display='inline' fontSize='input'>
+                    How did you hear about the Ecosystem Support Program?
+                  </PageText>
+                </FormLabel>
+
+                <Select
+                  id='howDidYouHearAboutESP'
+                  options={HOW_DID_YOU_HEAR_ABOUT_ESP_OPTIONS}
+                  onChange={onChange}
+                  components={{ DropdownIndicator }}
+                  placeholder='Select'
+                  closeMenuOnSelect={true}
+                  selectedOptionColor='brand.option'
+                  chakraStyles={chakraStyles}
+                />
+
+                {error && (
+                  <Box mt={1}>
+                    <PageText as='small' fontSize='helpText' color='red.500'>
+                      Referral source is required.
+                    </PageText>
+                  </Box>
+                )}
+              </FormControl>
+            )}
+          />
+
+          <FormControl id='referrals-control' mb={8}>
+            <FormLabel htmlFor='referrals' mb={1}>
+              <PageText fontSize='input'>
+                Did anyone recommend that you submit an application to the Ecosystem Support Program? If so, who?
+              </PageText>
+            </FormLabel>
+
+            <PageText as='small' fontSize='helpText' color='brand.helpText'>
+              Please submit the person&apos;s name only. This is not required.
+            </PageText>
+
+            <Input
+              id='referrals'
+              bg='white'
+              borderRadius={0}
+              borderColor='brand.border'
+              type='text'
+              h='56px'
+              mt={3}
+              position='relative'
+              color='brand.paragraph'
+              fontSize='input'
+              {...register('referrals', {
+                maxLength: 150
+              })}
+            />
+
+            {errors?.referrals?.type === 'maxLength' && (
+              <Box mt={1}>
+                <PageText as='small' fontSize='helpText' color='red.500'>
+                  Additional info cannot exceed 32768 characters.
```

Comment:
Should be `Referrals info cannot exceed 150 characters.`

Reply:
Good catch thanks
```suggestion
                  Referrals info cannot exceed 150 characters.
```